src/User32.Desktop/User32+DISPLAYCONFIG_ADDITIONAL_SINAL_INFO.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="DISPLAYCONFIG_VIDEO_SIGNAL_INFO"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        public struct DISPLAYCONFIG_ADDITIONAL_SINGAL_INFO
+        {
+            public uint bitvector1;
+
+            public uint VideoStandard
+            {
+                get
+                {
+                    return this.bitvector1 & 65535u;
+                }
+
+                set
+                {
+                    this.bitvector1 = value | this.bitvector1;
+                }
+            }
+
+            public uint VSyncFreqDivider
+            {
+                get
+                {
+                    return (this.bitvector1 & 4128768u) / 65536;
```

Comment:
It would be awesome if these values were encoded in hex instead of decimal. But that's just a wish.